packages/litqa/src/aviary/envs/litqa/env.py

```diff
@@ -151,6 +151,18 @@ async def step(
             truncated,
         )
 
+    async def get_id(self) -> str:
+        if (
+            isinstance(self._query, str)
+            or self._query.question_id
+            == MultipleChoiceQuestion.model_fields["question_id"].default
+        ):
+            raise ValueError(
+                "A multiple choice question with a non-default question ID was not"
+                " configured."
+            )
```

Comment:
This error isn't clear to me - why does a non-default ID lead to `get_id` failing?